rest-backend/src/main/java/com/sap/psr/vulas/backend/rest/SpaceController.java

```diff
@@ -226,6 +261,19 @@ else if(!new_space.hasNameAndDescription()) {
 				return new ResponseEntity<Space>(HttpStatus.NOT_FOUND);
 			}
 			
+			// Prevent modification of read-only spaces
+			try {
+				final Space managed_space = SpaceRepository.FILTER.findOne(this.spaceRepository.findBySecondaryKey(new_space.getSpaceToken()));
+				if(managed_space.isReadOnly()) {
+					log.error("Space [" + managed_space + "] is read-only");
```

Comment:
This means that once a space is set as read-only there is not way to go back. Are we sure this won't be required? 
If the read-only cannot be undone I would remove the possibility to create a new space with such flag

Reply:
I will add a check preventing the creation of workspaces with read-only enabled. This really does not seem to make any sense, hence, should be forbidden.